### ISSUE TITLE
LifetimeTracker is not working on Objective-C

### ISSUE THREAD

**Issue by dyegos** [krzysztofzablocki/LifetimeTracker#16]
When I installed it via CocoaPods and imported **@import LifetimeTracker** the only these two are exposed
```
FOUNDATION_EXPORT double LifetimeTrackerVersionNumber;
FOUNDATION_EXPORT const unsigned char LifetimeTrackerVersionString[];
```

I can't use anything else.

**Comment by jcislinsky** [comment]
I have prepared pull request fixing this.
https://github.com/krzysztofzablocki/LifetimeTracker/pull/27

**Comment by krzysztofzablocki** [comment]
This is now fixed in 1.3
